semcore/outside-click/src/OutsideClick.tsx

```diff
@@ -55,20 +63,53 @@ function OutsideClick(props: IFunctionProps<IOutsideClickProps>) {
   React.useEffect(() => {
     const outsideRoot = root ? getNodeByRef(root) : ownerDocument(nodeRef.current as any);
 
+    // disable previous events
+    if (outsideRoot instanceof HTMLElement && isModalRoot(outsideRoot)) {
+      OutsideClick.eventsMap.forEach(([root, events]) => {
```

Comment:
maybe `element` instead of `root` ?

Reply:
I used `rootElement` instead. I think it's important to state, that we operating on `root`, not just on any `element`. I think it make code more clear here. And that also keeps code in term `OutsideClick` component.